### PR TITLE
Contributors

### PR DESCRIPTION
**Pull request by WouterKvG**
Add functionality for saving contributors on goals. Depends on https://github.com/feedbackfruits/bepstore-api/pull/32.

### CODE REVIEW

**Comment by WouterKvG** [comment]
Don't mind the failing checks. CircleCi is not properly setup for engines yet.

**Comment by WouterKvG** [comment]
@steffansluis @bartist Please review!

File: app/models/goal.rb [hunk 1]
```diff
@@ -8,10 +8,12 @@ class Goal < Activity
   field :updateText, type: String
 
   # Core team
-  field :product_owner, type: Integer
-  field :head_design, type: Integer
-  field :head_frontend, type: Integer
-  field :head_backend, type: Integer
+  field :product_owner_id, type: User
```
Comment (steffansluis) [inline app/models/goal.rb]: Can this be done using relations? `has_one :product_owner, type: User` ?

Comment (WouterKvG) [inline app/models/goal.rb]: Will explore, but probably yes :)

**Pull request merged**